php.ini-development

```diff
@@ -517,6 +517,17 @@ report_memleaks = On
 ; This setting is on by default.
 ;report_zend_debug = 0
 
+; Set this to disable non-ASCII logging (more secure).
```

Comment:
Could you pls rewrite the whole comment in a little bit more formal way and remove the `(more secure)` bit as it is misleading  - it kind of says that using valid UTF-8 is less secure than ASCII which is not correct and even if it was, it should not be documented in here - it's really just for basic description IMHO.